src/misc.cpp

```diff
@@ -664,6 +664,7 @@ std::string getMD5(std::string data)
     return result;
 }
 
+// TODO: Add preprocessor option to disable (open web service safety)
```

Comment:
A complete scope limit should be implemented here to prevent (un)intentinally access of sensitive files on the server. Any ideas?